workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/Mono.TextTemplating/Mono.TextTemplating.csproj

```diff
@@ -8,6 +8,6 @@
     <DefineSymbols Condition="'$(TargetFramework)'!='netstandard2.0'">$(DefineSymbols);FEATURE_APPDOMAINS</DefineSymbols>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="System.CodeDom" Version="4.5.0" Condition="'$(TargetFramework)'=='netstandard2.0'" />
+    <PackageReference Include="System.CodeDom" Version="4.4.0" Condition="'$(TargetFramework)'=='netstandard2.0'" />
```

Comment:
Is this the other workaround which allows removing the `global.json` file?

Reply:
Yes - although as noted in the PR, need to test if this works on `2.3xx`

Reply:
Tested - it was broken. Fixed it 👍